streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/AccessTokenServiceImpl.java

```diff
@@ -99,10 +99,7 @@ public IPage<AccessToken> findAccessTokens(AccessToken tokenParam, RestRequest r
     @Override
     public boolean checkTokenEffective(Long userId, String token) {
         AccessToken res = baseMapper.getByUserToken(userId, token);
-        if (Objects.isNull(res) || AccessToken.STATUS_DISABLE.equals(res.getFinalStatus())) {
-            return false;
-        }
-        return true;
+        return !Objects.isNull(res) && !AccessToken.STATUS_DISABLE.equals(res.getFinalStatus());
```

Comment:
this code , We can write it better :
`return res != null && AccessToken.STATUS_ENABLE.equals(res.getFinalStatus());`